pymetacg/util.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include <Callgraph.h>
+#include <CgNode.h>
+#include <CgNodePtr.h>
+#include <MCGManager.h>
+
+using namespace metacg;
```

Comment:
Not sure if this applies here: But `using` in headers is (generally) dangerous.
Is there a reason that this file should not declare all of it inside `namespace metacg` and then have the `.cpp` file use that namespace?